Plugin~/WebRTCPlugin/DummyAudioDevice.cpp

```diff
@@ -46,13 +46,19 @@ namespace webrtc
             int64_t elapsed_time_ms = -1;
             int64_t ntp_time_ms = -1;
 
-            const int kBytesPerSample = 2;
-            const int kChannels = 2;
-            const int kSamplingRate = 48000;
-            const int kSamplesPerFrame = kSamplingRate * kFrameLengthMs / 1000;
-
-            char data[kBytesPerSample * kChannels * kSamplesPerFrame];
+            const size_t kBytesPerSample = 2;
+            const size_t kChannels = 2;
+            const size_t kSamplingRate = 48000;
+            const size_t kSamplesPerFrame = kSamplingRate * kFrameLengthMs / 1000;
+            std::vector<int16_t> audio_data(kChannels * kSamplesPerFrame);
+            void* data = audio_data.data();
```

Comment:
hum, could initialize std::vector in the constructor once? :)

Reply:
Yes, this buffer is able to be on constructer. 
I will fix it. Thank you for your review!

Reply:
fixed it